mockkubeapiserver/apiresourcelist.go

```diff
@@ -47,5 +48,19 @@ func (req *apiResourceList) Run(ctx context.Context, s *MockKubeAPIServer) error
 		}
 		response.APIResources = append(response.APIResources, resource)
 	}
+
+	// Return in a stable order, for more test predictability
```

Comment:
Does this sorting for consistent output diff check?

Reply:
Right :-)  I don't think it matters to real clients